Tests/Core/Formatter/SinglelineFormatterTests.swift

```diff
@@ -31,7 +32,15 @@ class SinglelineFormatterTests: XCTestCase {
         XCTAssertEqual(formatter.dictionaryString(keysAndValues: keysAndValues),
                        #"[1: "One", 2: "Two"]"#) // sorted
     }
-
+    
+    func testTupleString() {
+        assert(to: formatter.tupleString).expect([
+            when([], then: #"()"#),
+            when(["\"one\""], then: #"("one")"#),
+            when(["\"one\"", "\"two\""], then: #"("one", "two")"#),
```

Comment:
I suggest to include 2 types in tuple (e.g. `["\"one\"", "2"]`) that like representation of tuple

Reply:
That`s a good idea!

Reply:
modified in 6d7c912